spot_driver_cpp/src/kinematic/kinematic_component.cpp

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#include <rclcpp/node_options.hpp>
```

Comment:
This actually does need to be here for the component node library to build, but only because of a bug in rclcpp. [The fix for this bug has been backported to Humble](https://github.com/ros2/rclcpp/pull/2407) and will be available after the next sync.